lintrunner_adapters/adapters/refurb_linter.py

```diff
@@ -0,0 +1,164 @@
+"""Refurbish and modernize Python code"""
+
+from __future__ import annotations
+
+import argparse
+import logging
+import re
+import sys
+from textwrap import dedent
+
+from lintrunner_adapters import LintMessage, LintSeverity, add_default_options
+from lintrunner_adapters._common.lintrunner_common import run_command
+
+LINTER_CODE = "REFURB"
+
+RESULTS_RE = re.compile(
+    r"""(?mx)
+        ^
+        (?P<file>.*?):
+        (?P<line>\d+):
+        (?:(?P<column>-?\d+))?
+        (?:\s\[(?P<code>.*)\]:)?
+        \s(?P<message>.*)
+        $
+        """
+)
+
+
+def _test_results_re() -> None:
+    """Doctests.
+
+    >>> def t(s): return RESULTS_RE.search(s).groupdict()
+
+    >>> t(r"main.py:3:17 [FURB109]: Use `in (x, y, z)` instead of `in [x, y, z]`")
+    ... # doctest: +NORMALIZE_WHITESPACE
+    {'file': 'main.py', 'line': '3', 'column': '17', 'code': 'FURB109',
+     'message': 'Use `in (x, y, z)` instead of `in [x, y, z]`'}
+
+    >>> t(r"main.py:4:5 [FURB101]: Use `y = Path(x).read_text()` instead of `with open(x, ...) as f: y = f.read()`")
+    ... # doctest: +NORMALIZE_WHITESPACE
+    {'file': 'main.py', 'line': '4', 'column': '5', 'code': 'FURB101',
+     'message': 'Use `y = Path(x).read_text()` instead of `with open(x, ...) as f: y = f.read()`'}
+    """
+    pass
+
+
+def format_lint_message(message: str, code: str, show_disable: bool) -> str:
+    formatted = f"{message}\n"
```

Comment:
is it possible to move the newline to before `To disable, use...`?

Reply:
It already does that
![image](https://user-images.githubusercontent.com/43147888/213912497-324133f1-6ae4-447c-a69b-8fb031ed3586.png)

Reply:
So I wonder if there will be an extra newline when show_disable is False. But that’s ok I suppose? I don’t know if lintrunner will trim the final newline

Reply:
Added an extra newline to the disable message